maps/xtehr2zibs2024/Procedure-NusringIntervention.md

```diff
@@ -662,33 +662,34 @@ EHDSExclusive
 Seems to match
 
 
-## EHDSProcedure.outcome
+## EHDSProcedure.outcome (vs ZIBOutcomeOfCare)
 
 ### Table
 
 | attribute | value |
 |---|---|
 | xtehr | EHDSProcedure.outcome |
-| zib |  |
+| zib | ZIBOutcomeOfCare |
 | alias_zib |  |
 | binding_xtehr | {'strength': 'preferred', 'description': 'SNOMED CT'} |
 | card._xtehr | 0..1 |
-| card._zib |  |
+| card._zib | n/a |
 | definition_xtehr | The outcome of the procedure - did it resolve the reasons for the procedure being performed? |
-| definition_zib |  |
-| definitioncode_zib |  |
+| definition_zib | Root concept of the OutcomeOfCare information model. This root concept contains all data elements of the OutcomeOfCare information model. This consists of the HealthcareResult (The textual account of the healthcare result. If HealthcareResult cannot be entered as a functional/mental status, it can be described as free text in the healthcare result) or ZIBFunctionalOrMentalStatus (Description of the patient’s health condition in the form of a functional and/or mental status) |
+| definitioncode_zib | 423100009 SNOMED Results section |
 | id_xtehr | EHDSProcedure.outcome |
-| id_zib |  |
-| name_zib |  |
+| id_zib | NL-CM:13.4.1 |
+| name_zib | OutcomeOfCare |
 | path_xtehr | EHDSProcedure.outcome |
 | path_zib |  |
 | short_xtehr | The outcome of the procedure - did it resolve the reasons for the procedure being performed? |
 | stereotype_zib |  |
 | type_xtehr | CodeableConcept |
-| type_zib |  |
+| type_zib | ST and CD |
 
 ### Comments
-EHDSExclusive, seperate zib OutcomeOfCare evaluates the NursingIntervention and describes the determined status of the patient in terms of a certain problem. Comparing the healthcare result with the treatment objective provides insight into the effectivity of the nursing interventions/activities carried out for this problem. It is a part of the nursing process, together with nursing diagnoses/problems, the treatment objective and nursing interventions. 
+EHDSExclusive, seperate zib OutcomeOfCare evaluates the NursingIntervention and describes the determined status of the patient in terms of a certain problem. Comparing the healthcare result with the treatment objective provides insight into the effectivity of the nursing interventions/activities carried out for this problem. It is a part of the nursing process, together with nursing diagnoses/problems, the treatment objective and nursing interventions. In the zib the outcome is given in either textual format or structured in ZIBFunctionalOrMentalStatus.  
+Conclusion: ZIBOutcomeOfCare provides more detailed information about the outcome of a certain NursingIntervention compared to the EHDSProcedure.outcome, which only answers the question "did it resolve the reasons for the procedure being performed?"
```

Comment:
I'm not sure if I agree with the conclusion that the zib model provides more detailed information. The zib provides a functional or mental status as the outcome of a (nursing) procedure, and _if_ there is a treatment objective to compare it with, it's a powerful tool. It is not entirely known what the Xt-EHR model actually encodes, it could be a status, but it could also be a code for "partial success".

In any case, the models are different and as far as I can see incompatibke.